bin/client_package_standalone

```diff
@@ -29,7 +29,13 @@ DIR_BUILD="build/package"
 # Put all files to package directory (to avoid preserving directory structure)
 printf "Building Standalone package for OS '$OS' ..\n"
 DIR_TEMP=`mktemp -d ${DIR_BUILD}/${tempname}.XXXXXX`
-cp -vp ${BINARY} ${DIR_TEMP}/mysterium_client
+
+if [ "$OS" == "windows" ]; then
```

Comment:
I suggest simply appending .exe to target filename instead of duplicating copy command. I.e.: 
target_name = mysterium_client
if os == windows {
target_name = target_name + .exe
}
cp ..... $target_name

Reply:
Thanks, suggestion is nice to have, however I think we should not optimise here, because these are OS specific things and thus script is more readable

Reply:
OK, fair enough